app/Models/Campaign.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Rogue\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Campaign extends Model
+{
+    use SoftDeletes;
+
+    /**
+     * The attributes that should be mutated to dates.
+     *
+     * @var array
+     */
+    protected $dates = ['deleted_at'];
+
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = ['internal_title', 'start_date', 'end_date', 'created_at', 'updated_at'];
```

Comment:
Let's not make `created_at` or `updated_at` fillable here (since they're set automatically by Eloquent). We can set them explicitly (e.g. `$campaign->created_at = …`) if we want to keep these when importing campaigns via a script.

Reply:
oh oops - thanks @DFurnes ! Updated that!